rdtools/filtering.py

```diff
@@ -94,19 +97,19 @@ def clip_filter(power, quant=0.98):
         Boolean Series of whether the given measurement is below 99% of the
         quantile filter.
     '''
-    v = power.quantile(quant)
-    return (power < v * 0.99)
+    v = power_ac.quantile(quant)
+    return (power_ac < v * 0.99)
 
 
-def csi_filter(measured_poa, clearsky_poa, threshold=0.15):
+def csi_filter(poa_global_meas, poa_global_sim, threshold=0.15):
```

Comment:
I think this change reduces clarity, but it's consistent with pv-terms.

Reply:
maybe 'poa_meas' and 'poa_clearsky' ?

Reply:
Let's go with `poa_global_measured` and `poa_global_clearsky`